looker/definitions/ads.toml

```
@@ -139,22 +149,32 @@ from_expression = """
     ad_url,
     image_url,
     external_param,
-    impressions,
-    clicks,
-    click_rate,
-    dismisses,
-    dismiss_rate,
-    saves,
-    save_rate,
-    campaign_id,
-    spend
+    campaign_id
   FROM `moz-fx-data-shared-prod.ads.native_desktop_ad_metrics_hourly`
 )
 """
 friendly_name = "Native Desktop Ads Metrics Daily"
 description = """
   Tracks Native Desktop Ad Metrics Across Days
 """
-columns_as_dimensions = true
+columns_as_dimensions = false
```

```suggestion
columns_as_dimensions = true
```

I believe this should be `true`? The columns of this data source should show up in Looker, right?

---

Correct. Thanks for flagging.